packages/react-core/src/components/Truncate/__tests__/Truncate.test.tsx

```diff
@@ -28,3 +130,26 @@ test('renders middle truncation', () => {
   );
   expect(asFragment()).toMatchSnapshot();
```

Comment:
Can you refactor this test to not be a snapshot and explicitly assert against the default behavior when position=middle and traillingNumChars isn't passed?